CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 - Remove the `skip_glotpress` parameter from the `ios_bump_version_release` action [#443]
 - Remove the `ios_localize_project` and `ios_update_metadata` actions [#447]
+- Remove the `skip_deliver` parameter from the `ios_bump_version_hotfix` action
+- Remove the `skip_deliver` parameter from the `ios_bump_version_release` action
```

Comment:
tbh I think this could be combined in a single line  🙃
(nit that you might consider addressing once you'll re-target this PR to `iangmaia/trial` now that https://github.com/wordpress-mobile/release-toolkit/pull/447 has landed there)

```suggestion
- Remove the `skip_deliver` parameter from `ios_bump_version_hotfix` and `ios_bump_version_release` actions
```